fili-core/src/test/groovy/com/yahoo/bard/webservice/data/config/provider/FuzzyQueryMatcher.groovy

```diff
@@ -0,0 +1,135 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+
+package com.yahoo.bard.webservice.data.config.provider
+
+import com.yahoo.bard.webservice.data.metric.TemplateDruidQuery
+import com.yahoo.bard.webservice.druid.model.aggregation.Aggregation
+import com.yahoo.bard.webservice.druid.model.postaggregation.ArithmeticPostAggregation
+import com.yahoo.bard.webservice.druid.model.postaggregation.ConstantPostAggregation
+import com.yahoo.bard.webservice.druid.model.postaggregation.FieldAccessorPostAggregation
+import com.yahoo.bard.webservice.druid.model.postaggregation.PostAggregation
+
+/**
+ * Utility class to validate that TemplateDruidQueries have the same structure and values.
+ *
+ * It's ok if inner objects don't always have the same names.
+ */
```

Comment:
What are the situations where names are non-deterministic?  Filtered aggregations?